pkg/apis/kubeone/types.go

```diff
@@ -395,8 +399,27 @@ type ClusterNetworkConfig struct {
 
 	// KubeProxy config
 	KubeProxy *KubeProxyConfig `json:"kubeProxy,omitempty"`
+
+	// IPFamily "" | IPv4 | IPv6 | IPv4+IPv6.
+	IPFamily IPFamily `json:"ipFamily,omitempty"`
+
+	// NodeCIDRMaskSizeIPv4
+	NodeCIDRMaskSizeIPv4 *int `json:"nodeCIDRMaskSizeIPv4,omitempty"`
+
+	// NodeCIDRMaskSizeIPv6
+	NodeCIDRMaskSizeIPv6 *int `json:"nodeCIDRMaskSizeIPv6,omitempty"`
 }
 
+// IPFamily IPv4 | IPv6 | IPv4+IPv6.
+type IPFamily string
+
+const (
+	Unspecified IPFamily = "" // interpreted as IPv4
+	IPv4        IPFamily = "IPv4"
+	IPv6        IPFamily = "IPv6"
+	DualStack   IPFamily = "IPv4+IPv6"
```

Comment:
I would add `IPFamily` prefix to those variables, for example `IPFamilyIPv4`. See here for example: https://github.com/kubermatic/kubeone/blob/84756bf67a4f78f00adb243b494f4f50819cbd8e/pkg/apis/kubeone/types.go#L148-L160

Reply:
+1